src/OpenTracing/Formats.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace OpenTracing\Formats;
+
+/**
+ * Used a (single) arbitrary binary blob representing a SpanContext
```

Comment:
what is the format of the carrier for binary?

Reply:
I am not entirely sure. I'd say it is a `resource` but honestly I don't have too much experience with it. Any suggestion @felixfbecker @beberlei?

Reply:
It's probably just `string`. Strings in PHP are binary (not UTF8 like in JS) and are used for binary content. `resource` is for things like a file handle, a stream, a socket connection etc.